sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -224,7 +224,7 @@
         <dependency>
           <groupId>com.fasterxml.jackson.module</groupId>
           <artifactId>jackson-module-scala_2.10</artifactId>
-          <version>2.7.2</version>
+          <version>@jackson.version@</version>
```

Comment:
My original plan was to do it in a separate Jira, but I am happy to include it to this one. Will update the PR soon...